apps/docs/content/guides/auth/enterprise-sso/auth-sso-saml.mdx

```diff
@@ -415,3 +415,9 @@ Should you run into this problem, it is most likely a misconfiguration issue **o
 At this time it is not possible to extract the RSA private key used by your project's Supabase Auth server. This is done to keep the private key as secure as possible, given that SAML does not offer an easy way to rotate keys without disrupting service. (Please use a SAML 2.0 Metadata URL whenever possible for this reason!)
 
 If you really need access to the key, please [open a support ticket](https://supabase.com/dashboard/support/new) and we'll try to support you as best as possible.
+
+### Is multi-tenant SSO with SAML supported?
```

Comment:
Multi-tenant is somewhat loaded term, but this is what was used in the initial request. Open to alternative namings / rephrasings.